contracts/src/chainlink/cairo/ocr2/aggregator.cairo

```diff
@@ -1001,6 +1002,11 @@ func withdraw_funds{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_chec
     )
 
     let (link_due_uint256 : Uint256) = felt_to_uint256(link_due)
+    let (res) = uint256_le(link_due_uint256, balance)
+    with_attr error_message("Total amount due exceeds the balance"):
```

Comment:
We should prefix all error messages with contract/namespace like so:

```suggestion
    with_attr error_message("Aggregator: Total amount due exceeds the balance"):
```

Reply:
No need to do in this PR. I've created a separate issue for it: https://github.com/smartcontractkit/chainlink-starknet/issues/156